lib/settings.rb

```diff
@@ -0,0 +1,26 @@
+class Settings
+
+  CLASSIC = {
```

Comment:
It's good that the weapons are defined here as a constant and then referenced elsewhere effectively, helps keep your code DRY